src/main.tsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import store from './Redux/store';
 import App from '@/App.tsx';
 import '@/index.css';
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <Provider store={store}>
```

Comment:
Exchange lines 10 and 9.
ie: Line 10 should come first on line 9 and line 9 should be on line 10.
example:
`<React.StrictMode>
      <Provider store={store}>
            <App />
      </Provider>
</React.StrictMode>`